conf/base.config

```diff
@@ -10,9 +10,10 @@
 
 process {
 
-    cpus   = { check_max( 1    * task.attempt, 'cpus'   ) }
-    memory = { check_max( 6.GB * task.attempt, 'memory' ) }
-    time   = { check_max( 4.h  * task.attempt, 'time'   ) }
+    // TODO nf-core: Check the defaults for all processes
```

Comment:
I would just delete the TODO as you reviewed that point already when creating the pipeline.